Survey Test code (fixes #19)

waryhermit opened the pull request.

aberdean committed:
initial test

aberdean committed:
initial test

aberdean committed:
update url call

aberdean committed:
update url call

aberdean committed:
initial work on issue #13

aberdean committed:
initial work on issue #13

aberdean committed:
update test_resource.py

aberdean committed:
update test_resource.py

aberdean committed:
Merge branch '13_resource' of https://github.com/ole-vi/BeLL-Apps into 13_resource

aberdean committed:
remove test_member.py

aberdean committed:
update test_resource.py

aberdean committed:
add waiting time

aberdean committed:
sleeping is awesome

aberdean committed:
let's sleep

aberdean committed:
yet more sleep

aberdean committed:
Small corrections

aberdean committed:
add goto() and add_member()

aberdean committed:
add small reminder

aberdean committed:
fix indentation

aberdean committed:
test codeceptjs

aberdean committed:
fix typo

aberdean committed:
add dir

aberdean committed:
add sauce labs creds

aberdean committed:
fix typos

dogi committed:
Delete .travis.yml

dogi committed:
Delete test_successful_login.failed.png

dogi committed:
added codecept.conf.js and more (#50)

waryhermit committed:
helper function added (#55)

waryhermit committed:
Codecept Login with cookies + configs  (#63)

waryhermit committed:
survey code

waryhermit committed:
fixed some unwanted changes

waryhermit committed:
fixed #64

waryhermit committed:
Improvments to to survey test. Now fills out the survey and sends it.

waryhermit committed:
Merge branch 'dev' into test_survey_codecept

waryhermit committed:
Update survey_dialog.js

waryhermit committed:
Update saucelabs_helper.js

waryhermit committed:
Update saucelabs_helper.js

waryhermit committed:
test remove saucelabs stuff

waryhermit committed:
Update codecept.conf.js

waryhermit committed:
Update saucelabs_helper.js

waryhermit committed:
update

waryhermit committed:
add small wait

waryhermit committed:
another small wait

waryhermit committed:
another stupid wait

waryhermit committed:
more waits

waryhermit committed:
longer wait

waryhermit committed:
switch to checkOption, longer wait

waryhermit committed:
more ideas

dogi reviewed (changes requested):
there are 3 files which only change in whitespaces (one of them also doubles the comment line) and not to the better ;)
- [ ] `.travis.yml`
- [ ] `tests/bell.py`
- [ ] `tests/codeceptjs-init.js`

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update codecept.conf.js

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update bell.py

waryhermit committed:
Update bell.py

waryhermit committed:
Update codeceptjs-init.js

waryhermit committed:
restart travis

waryhermit committed:
Merge branch 'dev' into test_survey_codecept

waryhermit committed:
Update codecept.conf.js

waryhermit committed:
no change

waryhermit committed:
no change

waryhermit committed:
Merge branch 'test_survey_codecept' of https://github.com/waryhermit/BeLL-Apps into test_survey_codecept

waryhermit committed:
new changes

waryhermit committed:
use different locator

waryhermit committed:
Different click method

waryhermit committed:
prove that btn is clicked

waryhermit committed:
debug alert

waryhermit committed:
might fix it

waryhermit committed:
try 2

waryhermit committed:
add small wait

waryhermit committed:
revert router changes

waryhermit committed:
cleaning up pull request

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
Update .travis.yml

waryhermit committed:
remove debug

waryhermit committed:
sigh

waryhermit committed:
clean pull request more

waryhermit committed:
a little more cleanup

waryhermit commented:
@dogi I have addressed the issues you brought up and reduced the file changed count from 10 to 6. All the tests have passed for the last three commits. I think this is ready to be merged, but I want you to check it one last time first.

dogi reviewed (approved):
great ;)
